Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
As a general rule, I try to avoid doing anything in the hotfix branches which isn't strictly necessary - that's why I didn't backport the original `Directory.Build.props` change made to dev. It's a refactor/cleanup thing, which isn't exactly critical. Now, of course the risk is very low, but then there's not a huge amount to be gained here either :) I'll leave it up to you, I'm OK either way.

Reply:
This was actually spurred by some errors thrown while trying to update the packages via Rider (ordering of update issues), which led to updating the various files, which got......tedious. 

My thought is that if we plan to release at least one more patch after this (e.g. at least `2.1.4` and `2.1.5`) then it's worthwhile to move to the `dev`-style directory props.

That said, this was just some boilerplate from the branch that I was using to triage #623 and #651, so I don't really have an attachment either way

Reply:
Your call :) I'm OK with it either way...